Pear/Controller/HomeViewController.swift

```diff
@@ -317,20 +317,21 @@ extension HomeViewController: UNUserNotificationCenterDelegate {
                 // get rid of previously scheduled notifications
                 center.removeAllDeliveredNotifications()
                 center.removeAllPendingNotificationRequests()
-                self.scheduleNotifications(center: center, day: 2, hour: 8, title: "Meet your new pear!", body: "Set up this week's chat today 😊")
-                self.scheduleNotifications(center: center, day: 4, hour: 14, title: "Did you reach out yet?", body: "Choose a meeting time with your Pear before it's too late!")
-                self.scheduleNotifications(center: center, day: 6, hour: 12, title: "How's it going?", body: "New pairings will come out next week! ⌚️")
+                self.scheduleNotifications(center: center, day: 2, hour: 8, title: "Meet your new pear!", body: "Set up this week's chat today 😊", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 4, hour: 14, title: "Did you reach out yet?", body: "Choose a meeting time with your Pear before it's too late!", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 6, hour: 12, title: "How's it going?", body: "New pairings will come out next week! ⌚️", aboutPear: true)
+                self.scheduleNotifications(center: center, day: 0, hour: 0, title: "Are you running the latest version of Pear?", body: "Open TestFlight to check for new updates", aboutPear: false)
             }
         }
     }
 
-    private func scheduleNotifications(center: UNUserNotificationCenter, day: Int, hour: Int, title: String, body: String) {
+    private func scheduleNotifications(center: UNUserNotificationCenter, day: Int, hour: Int, title: String, body: String, aboutPear: Bool) {
         let content = UNMutableNotificationContent()
         content.title = title
         content.body = body
         content.sound = .default
         let dateComponents = DateComponents(hour: hour, minute: 0, second: 0, weekday: day)
-        let trigger = UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: true)
+        let trigger = aboutPear ? UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: true) : UNTimeIntervalNotificationTrigger(timeInterval: 1209600, repeats: true)
```

Comment:
```suggestion
        let trigger: UNCalendarNotificationTrigger = aboutPear ? .init(dateMatching: dateComponents, repeats: true) : .init(timeInterval: 1209600, repeats: true)
```
This is a little shorter.

Reply:
I would name and store the `timeInterval` of `1209600` in `Constants.swift` so its a less cryptic to what It is representing

Reply:
also break up the long line:
```
let trigger = aboutPear
  ? UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: true)
  : UNTimeIntervalNotificationTrigger(timeInterval: 1209600, repeats: true)

```